backend/database_handler/migration/versions/1ecaa2085aec_rollup_transactions.py

```diff
@@ -0,0 +1,40 @@
+"""rollup_transactions
+
+Revision ID: 1ecaa2085aec
+Revises: ab256b41602a
+Create Date: 2024-10-22 15:12:02.316347
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '1ecaa2085aec'
+down_revision: Union[str, None] = 'ab256b41602a'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('rollup_transactions',
```

Comment:
@kstroobants we also need to remove the audits table

Reply:
fixed in eca17e5fbf511c790d2025babe11cdf8c3bec6b7